sw/device/examples/sram_program/sram_program.h

```diff
@@ -0,0 +1,15 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_EXAMPLES_SRAM_PROGRAM_SRAM_PROGRAM_H_
+#define OPENTITAN_SW_DEVICE_EXAMPLES_SRAM_PROGRAM_SRAM_PROGRAM_H_
+
+#include <stdint.h>
+
+/**
+ * Type alias for the SRAM program entrypoint.
+ */
+typedef void sram_program_entrypoint(void);
```

Comment:
Is there ever any reason for returning a value from the SRAM program?  If so, should this be something like `uint32_t`?